src/main/java/org/tndata/android/compass/activity/ActionActivity.java

```diff
@@ -434,6 +440,62 @@ public void onClick(View v) {
     }
 
     /**
+     * Detect if the action belongs to a user and has a "datetime" external resource.
+     * See also: sendToCalendar.
+     *
+     * @return boolean
+     */
+    private boolean hasDatetimeExternalResource() {
+        if(isUserAction()) {
+            UserAction userAction = (UserAction) mAction;
+            String externalResource = userAction.getExternalResource();
+            String externalResourceName = userAction.getExternalResourceName();
```

Comment:
This might conflict with future attempts of reviving this feature, it may be a good idea to add an `external_resource_type` to the action and include types there.